Helpers.gs

```diff
@@ -359,10 +359,20 @@ function createEvent(event, calendarTz){
   else if (event.hasProperty('summary'))
     newEvent.summary = icalEvent.summary;
 
-  if (addOrganizerToTitle && event.hasProperty('organizer')){
-    var organizer = event.getFirstProperty('organizer').getParameter('cn').toString();
-    if (organizer != null)
-      newEvent.summary = organizer + ": " + newEvent.summary;
+  if (event.hasProperty('organizer')){
+    var organizerName = event.getFirstProperty('organizer').getParameter('cn');
+    var organizerMail = event.getFirstProperty('organizer').getParameter('mailto');
+    newEvent.organizer = callWithBackoff(function() {
```

Comment:
Is there a reason this needs to be inside `callWithBackoff`? Did you see this fail intermittently?

Reply:
I have not and i actually don't know if it **can** fail. That's a 'better safe than sorry' solution.